migrator/zenodo_rdm_migrator/extract/kafka.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Migrator extraction classes for multiple sources (PGSQL, JSONL files)."""
+
+from invenio_rdm_migrator.extract import Extract
+from kafka import KafkaConsumer
+
+
+class KafkaExtract(Extract):
```

Comment:
isn't this generic enough to be in invenio_rdm_migrator?

Reply:
Yes... The Kerberos auth part (i.e. installing the `gssapi` Python library) is the only specific thing we need for ZenodoRDM. Though I have a feeling we will be the only ones to ever use the Kafka/Debezium pipeline for migration.